src/bin/nydus-image/main.rs

```diff
@@ -348,6 +350,49 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                 )
         );
 
+    let app = app.subcommand(
+            App::new("chunkdict")
+                .about("deduplicate RAFS filesystem metadata")
+                .subcommand(
+                    App::new("save")
+                        .about("Save chunk info to a database")
+                        .arg(
+                            Arg::new("bootstrap")
+                            .short('B')
+                            .long("bootstrap")
+                            .help("File path of RAFS meta blob/bootstrap")
+                            .conflicts_with("BOOTSTRAP")
+                            .required(false),
+                        )
+                    .arg(
+                        Arg::new("database-path")
+                            .long("database-path")
+                            .help("File path of metadata database, like: 'sqlite:///path/to/database.db'")
```

Comment:
Database connection URI for assisting chunk dict generation, e.g. `sqlite:///path/to/database.db`.